frontend/src/pages/accounts/profile.page.tsx

```diff
@@ -255,9 +303,19 @@ const Profile: NextPage = () => {
             <dt className={styles.label}>
               {l10n.getString("profile-stat-label-aliases-used-2")}
             </dt>
-            <dd className={styles.value}>
-              {numberFormatter.format(allAliases.length)}
-            </dd>
+            {allAliases.length >= freeMaskLimit &&
+            !profile.has_premium &&
+            isPhonesAvailableInCountry(runtimeData.data) ? (
+              <MaxedMasksTooltip>
+                <dd className={`${styles.value} ${styles.maxed}`}>
+                  {numberFormatter.format(allAliases.length)}
+                </dd>
+              </MaxedMasksTooltip>
```

Comment:
I think this will [result in invalid HTML](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl):

> **Permitted content** Either: Zero or more groups each consisting of one or more `<dt>` elements followed by one or more `<dd>` elements, optionally intermixed with           `<script>` and `<template>` elements. Or: (in WHATWG HTML, W3C HTML 5.2           and later) One or more `<div>` elements, optionally intermixed with `<script>` and           `<template>` elements.

Would it be possible to swap them around?

```suggestion
              <dd className={`${styles.value} ${styles.maxed}`}>
                <MaxedMasksTooltip>
                  {numberFormatter.format(allAliases.length)}
                </MaxedMasksTooltip>
              </dd>
```